package.json

```diff
@@ -32,13 +34,14 @@
     "stylelint": "^7.6.0"
   },
   "scripts": {
-    "ava": "ava --verbose \"__tests__/**/*.js\"",
+    "jest": "jest",
```

Comment:
Done in https://github.com/stylelint/stylelint-config-standard/pull/50/commits/f73e8e8bdf787749bbbdf54ed96a6ec05a9da65b (I originally had cache issues, and was using `jest --no-cache` here)